frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewLayoutSet/CreateNewLayoutSet.module.css

```diff
@@ -0,0 +1,9 @@
+.savelayoutSetButton {
+  display: flex;
+  align-self: flex-start;
+  border: 2px solid var(--success-color);
```

Comment:
Could a designsystem sizing variable be used here instead of 2px?
For example: `--fds-sizing-1`

Reply:
I tried it but it gave me a thick border, what do you think?

Reply:
It might actually make sense to use just 2px here 👍